### PR TITLE
Features/#20

### PR DESCRIPTION
**Pull request by eskabetxe**
### Description of the Change

- upgraded kafka to last version
- solve some test problems

### Applicable Issues
https://github.com/idealista/kafka-role/issues/20
https://github.com/idealista/kafka-role/issues/26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ These instructions will get you a copy of the role for your ansible playbook. On
 
 ### Prerequisities
 
-Ansible 2.2.1.0 version installed.
+Ansible 2.4.2.0 version installed.
```
Comment (jnogol) [inline README.md]: 2.4.2.0 or 2.4.0.0? In `travis.yml` and `meta/main.yml` is 2.4.0.0

File: tests/requirements.yml [hunk 2]
```diff
@@ -1,8 +1,13 @@
 ---
 - src: idealista.java-role
-  version: 2.0.0
+  version: 3.0.0
   name: java
 
-- src: idealista.zookeeper-role
-  version: 1.1.0
-  name: zookeeper
+#- src: idealista.zookeeper-role
```
Comment (jnogol) [inline tests/requirements.yml]: Guess we have to release new zookeeper role version, haven't we?

File: README.md (hunk 1 above)
Comment (eskabetxe) [reply README.md]: fixed

File: tests/requirements.yml (hunk 2 above)
Comment (dortegau) [reply tests/requirements.yml]: Although it should not be done, please do not leave commented lines

**Comment by dortegau** [comment]
```
TASK [zookeeper : ZOOKEEPER | Untar zookeeper] *********************************
fatal: [zookeeper.vm]: FAILED! => {"changed": false, "failed": true, "msg": "Source '/tmp/ansible_w3LsMx/zookeeper-3.4.12.tar.gz' not readable"}
```

https://travis-ci.org/idealista/kafka-role/builds/389623460#L1096

**Pull request merged**